pages/settings/sessions.vue

```diff
@@ -44,14 +63,55 @@ definePageMeta({
 })
 
 useHead({
-  title: 'Sessions - Modrinth',
+  title: () => `${formatMessage(messages.sessionsTitle)} - Modrinth'`,
```

Comment:
You appear to be using `formatMessage` here before it's actually defined at line 75. While this may work, it's looks confusing, and I'd recommend moving the destructuring of `formatMessage` above this `useHead` call.